app/schemas/user.py

```
@@ -0,0 +1,32 @@
+from pydantic import BaseModel, EmailStr, Field
+from typing import Optional
+from enum import Enum
+from pydantic import ConfigDict
+
+class UserType(str, Enum):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)